features/step_definitions/steps.rb

```diff
@@ -17,14 +18,22 @@
 end
 
 Given(/^a page with the HTML:$/) do |string|
-  @page = Capybara.string(string.to_s)
+  WebServer.ensure_running_on(54321)
+  WebServer.write_page "scenario.html", "<html><body>#{string}</body></html>"
+  browser.visit 'http://localhost:54321/scenario.html'
```

Comment:
Only if there was a good reason. You aren't allowed to say "hard coding" :tongue: